test/unit/pipeline/test_pipeline_train_model.py

```diff
@@ -316,7 +315,8 @@ def test_train_model_InputDataInsufficientError(
                 pj=self.pj,
                 input_data=self.train_input,
                 check_old_model_age=False,
-                trained_models_folder="./test/unit/trained_models",
+                mlflow_tracking_uri="./test/unit/trained_models/mlruns",
+                trained_models_folder="./test/unit/trained_models/mlruns",
```

Comment:
I see a lot of duplications of mlflow_tracking_uri and trained_models_folder. 
Maybe we can have a short call on how they relate to each other and why we need both.

Reply:
See comments above or if call is nicer we can do that. Will fix the duplication where it is possible. We only need the duplication of mlflow_tracking_uri/trained_models_folder for removing the trained_models in persistent volume